src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,35 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment,
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  comment: {
+    name,
+    email,
+    body,
+  },
+}) => (
+  <>
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{ name }</strong>
+
+        {' by '}
+
+        <a
+          className="CommentInfo__email"
+          href={`{mailto:${email}`}
```

Comment:
```suggestion
          href={`mailto:${email}`}
```
one test failed.
looks like problem is here - extra and redundant curly brace
![Screenshot from 2023-06-10 15-39-25](https://github.com/mate-academy/react_static-list-of-posts/assets/33029864/d6b96e56-c3f2-4966-a1e1-63b50dbaee8c)